easybuild/easyblocks/g/gromacs.py

```diff
@@ -324,34 +378,61 @@ def configure_step(self):
                     if not regex.search(out):
                         raise EasyBuildError("Pattern '%s' not found in GROMACS configuration output.", pattern)
 
+    def build_step(self):
+        """
+        Custom build step for GROMACS; Skip if CUDA is enabled and the current
+        iteration is for double precision
+        """
+
+        if self.is_double_precision_cuda_build():
+            self.log.info("skipping build step")
+            return
+
+        super(EB_GROMACS, self).build_step()
+
     def test_step(self):
         """Run the basic tests (but not necessarily the full regression tests) using make check"""
+
+        if self.is_double_precision_cuda_build():
+            self.log.info("skipping test step")
+            return
+
         # allow to escape testing by setting runtest to False
-        if not self.cfg['runtest'] and not isinstance(self.cfg['runtest'], bool):
+        if self.cfg['runtest'] is None or self.cfg['runtest']:
```

Comment:
Good catch fixing this. Just a note that this changes behavior. Previously it would be executed only if `runtest=""` or `runtest=None` which is very weird.

Reply:
yes it does change behavior, but at least now it is correct.